app/views/categories/show.html.erb

```diff
@@ -2,19 +2,25 @@
   <h1><%= @category.name %></h1>
 
   <div class="row">
-    <% @projects.each do |project| %>
-    <div class="box col-lg-3">
-      <%= link_to("#{project.title}", tenant_project_path(slug: project.tenant.slug, id: project.id))%>
-      <div class="thumbnail">
-        <%= image_tag project.photos.first.image.url %>
+    <% if @projects.any? %>
+      <% @projects.each do |project| %>
+      <div class="box col-lg-3">
+        <%= link_to("#{project.title}", tenant_project_path(slug: project.tenant.slug, id: project.id))%>
+        <div class="thumbnail">
+          <%= image_tag project.photos.first.image.url %>
+        </div>
+        <h5 class="project-desc"><%= project.description %></h5>
+        <%= form_for :pending_loan, url: pending_loan_path do |f| %>
+          <%= f.hidden_field :project_id, value: project.id %>
+          <%= f.hidden_field :loan_amount, value: project.price %>
+          <%= f.submit 'Lend', class: "btn btn-default" %>
+        <% end %>
       </div>
-      <h5 class="project-desc"><%= project.description %></h5>
-      <%= form_for :pending_loan, url: pending_loan_path do |f| %>
-        <%= f.hidden_field :project_id, value: project.id %>
-        <%= f.hidden_field :loan_amount, value: project.price %>
-        <%= f.submit 'Lend', class: "btn btn-default" %>
       <% end %>
-    </div>
+    <% else %>
+      <div class="box col-lg-3">
+        There are no projects for this category
+      </div>
     <% end %>
   </div>
 </div>
```

Comment:
I think we can delete this whole file, right? None of our user stories ask us have a show page for each category.